src/ui/components/Transcript/Transcript.tsx

```diff
@@ -45,16 +46,22 @@ function Transcript({ recordingId, pendingComment }: PropsFromRedux) {
       </div>
       <div className="transcript-list flex-grow overflow-auto overflow-x-hidden flex flex-col items-center bg-white h-full">
         {displayedComments.length > 0 ? (
-          <div className="p-2 overflow-auto w-full flex-grow space-y-2">
+          <div className="p-3 overflow-auto w-full flex-grow space-y-3">
             {sortBy(displayedComments, ["time"]).map(comment => {
               return <CommentCard comment={comment} key={"id" in comment ? comment.id : 0} />;
             })}
           </div>
         ) : (
-          <div className="transcript-list p-2 self-stretch space-y-4 text-lg text-gray-500">
-            {isAuthenticated
-              ? "None yet! Please click the video to add a comment."
-              : "Please log in to add a comment to this replay."}
+          <div className="transcript-list p-4 self-stretch space-y-4 text-lg text-gray-500 onboarding-text">
+            <MaterialIcon className="forum large-icon">forum</MaterialIcon>
+            <h2>{isAuthenticated ? "Start a conversation" : "Sign in to get started"}</h2>
+
+            <p>
+              {isAuthenticated
+                ? "Add a comment to the video, a line of code, or (soon!) a console message."
+                : "Once signed in, you can add comments and make your voice heard!"}
+            </p>
+            <img src="/images/comment-onboarding-arrow.svg" class="arrow" />
```

Comment:
I think typescript is complaining because`class` should be `className`

Reply:
This is a mistake I keep making. Thanks! I'll soon learn this lesson for good!